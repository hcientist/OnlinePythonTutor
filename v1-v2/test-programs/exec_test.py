exec "import os; os.system('echo security breach')"
