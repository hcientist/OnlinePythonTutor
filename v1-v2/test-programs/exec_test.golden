[ { "event" : "step_line",
    "func_name" : "<module>",
    "globals" : {  },
    "line" : 1,
    "stack_locals" : [  ],
    "stdout" : ""
  },
  { "event" : "call",
    "func_name" : "<module>",
    "globals" : {  },
    "line" : 1,
    "stack_locals" : [  ],
    "stdout" : ""
  },
  { "event" : "step_line",
    "func_name" : "<module>",
    "globals" : {  },
    "line" : 1,
    "stack_locals" : [  ],
    "stdout" : ""
  },
  { "event" : "exception",
    "exception_msg" : "ImportError: __import__ not found",
    "func_name" : "<module>",
    "globals" : {  },
    "line" : 1,
    "stack_locals" : [  ],
    "stdout" : ""
  }
]
